manuscript/Makefile

```diff
@@ -66,6 +67,16 @@ wc:
 lint:
 	@ proselint $(PROJECT).tex
 
+# Create greyscale PDF version
+greyscale: $(OUTPUT)/$(PROJECT).pdf
+    gs \
+   -sDEVICE=pdfwrite \
+   -sProcessColorModel=DeviceGray \
+   -sColorConversionStrategy=Gray \
+   -dOverrideICC \
+   -o $(OUTPUT)/$(PROJECT)-greyscale.pdf \
+   -f $(OUTPUT)/$(PROJECT).pdf
```

Comment:
@santis19 sorry, I didn't see this until now. I need to turn on email notifications again.

The PDF is the `$(OUTPUT)` variable. Try this instead:

```
greyscale: $(OUTPUT)
    gs \
   -sDEVICE=pdfwrite \
   -sProcessColorModel=DeviceGray \
   -sColorConversionStrategy=Gray \
   -dOverrideICC \
   -o $(patsubst %.pdf,%-greyscale.pdf,$<) \
   -f $<
```

Reply:
You're right @leouieda, I've carried the $(OUTPUT)/$(PROJECT).pdf syntax from the old Makefile.
I've just fixed it.